File: sql/plsql/Python3/PlSqlBaseParser.py
from antlr4 import *

class PlSqlBaseParser(Parser):

    _isVersion10 = False
    _isVersion12 = True

    def isVersion10(self):
        return self._isVersion10

    def isVersion12(self):
        return self._isVersion12

    def setVersion10(self, value):
        self._isVersion10 = value

    def setVersion12(self, value):
        self._isVersion12 = value

File: query/sql/plsql/Go/plsql_base_parser.go
package parser

import (
    "github.com/antlr/antlr4/runtime/Go/antlr"
)

// PlSqlBaseParser implementation.
type PlSqlBaseParser struct {
    *antlr.BaseParser
    _isVersion12 bool
    _isVersion10 bool
    self PlSqlParserBase
}

/*
PlSqlParserBase(TokenStream input) {
    super(input);
    self = this;
}

func isVersion12() bool {
    return _isVersion12;
}

func setVersion12(boolean value) {
    _isVersion12 = value;
}

func isVersion10() func {
    return _isVersion10;
}

func setVersion10(boolean value) {
_isVersion10 = value;
}
*/